README.md

```diff
@@ -1,4 +1,4 @@
-# mmctl ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mmctl/master.svg)
+# mmctl ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mmctl/master.svg)[![Go Report Card](https://goreportcard.com/badge/github.com/mattermost/mmctl)](https://goreportcard.com/report/github.com/mattermost/mmctl)
```

Comment:
very small nit: can you please put a space between badges?

![image](https://user-images.githubusercontent.com/223323/67084117-a069ef00-f19c-11e9-821b-ae110c9bb268.png)

Reply:
@mgdelacroix sure, updated